.env.dist

```diff
@@ -31,7 +31,7 @@ DRUPAL_ACCOUNT_PASSWORD=admin
 DRUPAL_DATABASE_HOST=mysql
 DRUPAL_DATABASE_PORT=3306
 DRUPAL_DATABASE_USERNAME=root
-DRUPAL_DATABASE_PASSWORD=
+DRUPAL_DATABASE_PASSWORD=""
```

Comment:
Wow is this the fix?? I assumed it would be something very complicated :+1:

Reply:
No, unfortunately, it's more problematic.